pkgs/tools/networking/bgpdump/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchzip, autoconf, zlib, bzip2 }:
+
+stdenv.mkDerivation rec {
+  name = "bgpdump-94a0e724b335";
+
+  src = fetchzip {
+    url = "https://bitbucket.org/ripencc/bgpdump/get/94a0e724b335.zip";
+    sha256 = "09g9vz2zc4nyzl669w1j7fxw21ifja6dxbp0xbqh6n7w3gpx2g88";
+  };
+
+  buildInputs = [ autoconf zlib bzip2 ];
+  configurePhase = ''
+    autoheader
+    autoconf
+    ./configure --prefix=$out
+  '';
+
+  meta = {
+    homepage = https://bitbucket.org/ripencc/bgpdump/wiki/Home;
+    description = ''Analyze dump files produced by Zebra/Quagga or MRT'';
+    license = stdenv.lib.licenses.free;
```

Comment:
I think we can put `license = stdenv.lib.licenses.hpnd` for this, the license looks largely the same and is only missing two small sentences at the end of the first paragraph about warranty and suitability, which is what the second paragraph is about anyway.